aws/templates/docker_compose.tftpl

```diff
@@ -33,6 +33,8 @@ services:
       DISABLE_READ_API: "true"
       DISABLE_WRITE_API: "true"
 %{ endif ~}
+      BLOCKSCOUT_PROTOCOL: "https"
+      BLOCKSCOUT_HOST: "testnet.explorer.omni.network/"
```

Comment:
Do we want to parametrize this both for staging and testnet?

Why a forward slash at the end? Per docs, default value is localhost (without/)

Reply:
Yeah that's probably the right way to do it

I used the slash locally, but just tested again, and it worked without it too. So I'll remove it for cleanliness